File: QuizApp/src/QuizApp.BLL/Dto/PassingTest/CreatedTestResultDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace QuizApp.BLL.Dto.PassingTest
{
    public class CreatedTestResultDto
    {
        public int Id { get; set; }

        public string IntervieweeName { get; set; }

        public DateTime PassingStartTime { get; set; }

        public DateTime PassingEndTime { get; set; }

        public double Score { get; set; }


        public ICollection<CreatedResultAnswerDto> ResultAnswers { get; set; } = new List<CreatedResultAnswerDto>();
    }
}

File: frontend/src/app/models/passing-test/created-test-result-dto.ts
import { CreatedResultAnswerDto } from './created-result-answer-dto';

export interface CreatedTestResultDto {
    id: number;
    intervieweeName: string;
    passingStartTime: Date;
    passingEndTime: Date;
    score: number;

    resultAnswers: CreatedResultAnswerDto[];
}
